Form-Controls/index.html

```diff
@@ -1,28 +1,94 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
+    <main>
+      <section>
+        <div class="product-form">
+          <form>
+            <!-- write your html here-->
+            <!-- try writing out the requirements first-->
+            <div>
+              <label for="name">Enter Name:</label><br />
+              <input
+                type="text"
+                id="name"
+                name="name"
+                pattern="[a-zA-Z\s]+"
+                minlength="2"
+                maxlength="15" 
+                required
+              /><br />
+              <label for="email">Enter Email:</label><br />
+              <input
+                type="email"
+                id="email"
+                name="email"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
+                required
+              /><br />
+            </div>
 
-</html>
\ No newline at end of file
+            <div>
```

Comment:
Is there another element you could have chosen to use instead? Why does it matter how we group form controls?

https://www.w3.org/WAI/tutorials/forms/grouping/